cou/steps/execute.py

```diff
@@ -18,45 +18,24 @@
 import sys
 
 from aioconsole import ainput
-from colorama import Style
 
 from cou.steps import ApplicationUpgradePlan, BaseStep, UpgradeStep
 from cou.utils import progress_indicator
+from cou.utils.text_styler import bold, normal
 
 AVAILABLE_OPTIONS = ["y", "n"]
 
 logger = logging.getLogger(__name__)
 
 
-def prompt(parameter: str) -> str:
+def prompt_message(parameter: str) -> str:
     """Generate eye-catching prompt.
 
     :param parameter: String to show at the prompt with the user options.
     :type parameter: str
     :return: Prompt string with the user options.
     :rtype: str
     """
-
-    def bold(text: str) -> str:
```

Comment:
**question:** 

I saw a small refactor here. Can you provide more information for this change?

Reply:
This PR is based on #196 and #197. The information about this refactoring is mentioned in #196: Move text styling code from steps/execute.py to a newly created utils/text_styler.py to be shared by multiple modules.